.github/workflows/release-images.yml

```diff
@@ -0,0 +1,50 @@
+name: Release images.
+
+on:
+  push:
+    branches:
+      - "master"
+      - "staging"
+
+env:
+  SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+
+jobs:
+  tests:
+    uses: Quadrabee/arnavon/.github/workflows/integration.yml@master
```

Comment:
I think it would be better to use relative path so that a PR can even refactor the workflows: `- uses: ./.github/workflows/...`

Reply:
@llambeau i'm not sure this is the same. Here we're calling a reusable workflow, different from calling an action. [https://docs.github.com/en/actions/learn-github-actions/reusing-workflows#example-reusable-workflow]()